### PR TITLE
Address Travis configuration warnings and lint 

### PR DESCRIPTION
**Pull request by cottsay**
Warnings:
* `root`: deprecated key `ros_gpg_key_add_string` (anchor on a non-private key)
* `jobs.include`: deprecated key `sudo` (The key `sudo` has no effect anymore.)

Infos:
* `root`: missing `os`, using the default `linux`
* `root`: missing `language`, using the default `ruby`
* `root`: key `matrix` is an alias for `jobs`, using `jobs`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,20 +39,20 @@ env:
     - OS_CODE_NAME2=bionic
     - ARCH=amd64
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+os: linux
 dist: xenial
-matrix:
+language: python
```
Comment (dirk-thomas) [inline .travis.yml]: With this the `language` key can probably be removed from each jobs section.

File: .travis.yml [hunk 2]
```diff
@@ -39,20 +39,20 @@ env:
     - OS_CODE_NAME2=bionic
     - ARCH=amd64
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+os: linux
```
Comment (dirk-thomas) [inline .travis.yml]: Why should we specify the default explicitly? Just for readability?

Comment (cottsay) [reply .travis.yml]: Yeah, and to squelch the extra info being generated for each build. I can drop it if you think it makes things worse somehow.

File: .travis.yml (hunk 1 above)
Comment (cottsay) [reply .travis.yml]: 7fd7982

File: .travis.yml (hunk 2 above)
Comment (dirk-thomas) [reply .travis.yml]: It can stay - I was just making sure I understand it correctly.

**Pull request merged**